test/widgets/new_dm_sheet_test.dart

```diff
@@ -254,17 +254,17 @@ void main() {
   });
 
   group('user selection', () {
-    void checkUserSelected(WidgetTester tester, User user, bool expected) {
-      final icon = tester.widget<Icon>(find.descendant(
-        of: findUserTile(user),
-        matching: find.byType(Icon)));
+      Finder findInUserTile(User user, Finder finder) => find.descendant(of: findUserTile(user), matching: finder);
```

Comment:
nit: separately, this is too wide: necessary information appears after 80 columns.